services/github/github-gist-last-commit.tester.js

```diff
@@ -0,0 +1,36 @@
+import dayjs from 'dayjs'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('last commit in gist (recent)')
+  .get('/7e188c35fd5ca754c970e3a1caf045ef.json')
+  .intercept(nock =>
+    nock('https://api.github.com')
+      .get('/gists/7e188c35fd5ca754c970e3a1caf045ef')
+      .reply(200, {
+        updated_at: dayjs(),
+      })
+  )
+  .expectBadge({ label: 'last commit', message: 'today', color: 'brightgreen' })
+
```

Comment:
I hate to mention this since I'm sure you had fun battling nock :smile: but I think we should drop this test.

We mostly use our service tests as an integration and e2e style test suite to both give us some high level assurances about the service and routes, but particularly the touch points with the hundreds of backend APIs we integrate with (and which can change without notice).

In cases where we have service class with non trivial `render` or `transform` functions we will add more standard unit tests to validate those, but we typically refrain from doing so when those are simple functions or just using some of our standard helpers like this one is (those helpers are extensively tested elsewhere).

Reply:
> I hate to mention this since I'm sure you had fun battling nock 😄 but I think we should drop this test.

Haha, a little bit :D

Sure, removed this test :)